### ISSUE TITLE
Xulrunner 35.0.1 + Support

### ISSUE THREAD

**Issue by jmercouris** [smazga/conkeror_mac_bundler#6]
Hey, I've noticed that this seems to only support compilation up to XUL 35.0.1, since the releases are 40.0+ now, are there any plans to release it? Any way to help contribute to get this done?

**Comment by smazga** [comment]
Versions of XUL past 35 won't start with the layout that we create; there are files missing or something. I don't know XUL, and I can't find any decent documentation on it, and there seem to be breaking changes (as far as conkeror_mac_bundler is concerned) with almost every release, so I decide to stop chasing the upgrade train. Plus, I'm not sure what a new XUL even buys us in the conkeror context. It seems to work with 35, so I figured I'd leave it there until it stops working.

As far as helping to get it to a newer version? If you know the magical incantation to get it to work with a newer XUL, I'm happy to accept a patch or even just an explanation on how to do it.

**Comment by jmercouris** [comment]
I wish I knew what the magical incantation was, I was playing around with newer versions of XUL, and I actually launched Conkeror with the latest Firefox XUL via the '/Applications/Firefox.app -app’ trick, but it was unbelievably slow and would crash all the time.

Not sure what the problem is, on a kind of unrelated note, I did take a look at QuteBrowser but it just does not have the same features/bindings.

**Comment by Wilfred** [comment]
The downside is that conkeror doesn't have the security fixes in most recent releases. That said, https://ftp.mozilla.org/pub/xulrunner/releases/ only goes up to 41.0 anyway.

Would it be possible to use Firefox.app with conkeror_mac_bundler?

**Comment by jmercouris** [comment]
This is true, and it is possible, I've done it with the latest version of firefox on my system, but it was unbelievably slow and kept crashing
